dart/biomechanics/DynamicsFitter.cpp

```diff
@@ -10528,9 +10529,9 @@ void DynamicsFitter::zeroLinearResidualsOnCOMTrajectory(
           int timestepsSinceDv = t - missingIndex;
           int mapping = missingStepMappings[j];
 
-          int missingXCol = 6 + mapping * 3;
-          int missingYCol = 6 + mapping * 3 + 1;
-          int missingZCol = 6 + mapping * 3 + 2;
+          int missingXCol = 6 + (numForcePlates * 3) + mapping * 3;
```

Comment:
This was just a straightforward bug in the previous version, unrelated to the current fix.